### PR TITLE
Update letter for Huntsville, AL

### LINKED ISSUE #664: Revision needed on Huntsville - AL
Email template is generic and includes no local information, should be expanded on per the style guide.

### PR DESCRIPTION
**Pull request by jjs451**
Closes #664

### CODE REVIEW

**Comment by bingbongle** [comment]
@averymcguirt this doesn't seem to have gone through editorial flow via issues. Can you take a look and triage? Ideally PRs wouldn't be made like this with content changes. Feel free to close if you don't think this should be merged.

**Comment by jjs451** [comment]
If email templates aren't supposed to be directly PR'd, the team may want to add something to the readme to make that clearer.  I went to the repo, saw the email style guide at the bottom of the readme and assumed I was supposed to fork and PR.

**Comment by bingbongle** [comment]
@jjs451 yup, we're working on making changes to the worflow. Keep in mind the project has only been live a couple of days, and we're completely in over our heads, so only doing what we can! If you check out TEAMS.md in the root directory, there's a bit more of an explanation about how the current workflow works.

File: _emails/us/alabama/huntsville.md [hunk 1]
```diff
@@ -11,9 +11,16 @@ recipients:
 - bill.kling@huntsvilleAL.gov
 - Will.culver@huntsvilleAL.gov
 - strongalabama@att.net
+- Contact@HuntsvilleAL.gov
 subject: Defund Huntsville PD
 body: |
-  Hi, my name is [NAME] and I am a resident of [NEIGHBORHOOD]. I am asking to redirect money away from the Huntsville PD and into the following social services: [LIST - Community Development, COVID Relief, Education, Parks & Recreation]. I request that an emergency meeting be called before the fiscal year goes into effect in order to deny the mayor's proposed budget and reallocate these funds to resources from which the citizens can benefit.
+  Hi, my name is [NAME] and I am a resident of [NEIGHBORHOOD]. I am writing to demand that the Huntsville City Council adopt a city budget that prioritizes community wellbeing, and redirects funding away from the police.
+  
+  The proposed budget for 2020 allocates 31.7% (50,541,228) of the personnel budget towards police.  This is more than the next highest two budget categories combined (Fire & Rescue and Public Works).
```
Comment (serifluous) [inline _emails/us/alabama/huntsville.md:19]: Budget confirmed here: https://3jzi0q2zthm01oqpx2h96lz1-wpengine.netdna-ssl.com/wp-content/uploads/2019/09/Proposed-FY20-Budget-Exhibit-A.pdf

**Pull request merged**